File: src/com/Torvald/Terrarum/Actors/CanBeStoredAsItem.java
package com.Torvald.Terrarum.Actors;

import com.Torvald.Terrarum.GameItem.InventoryItem;

/**
 * Created by minjaesong on 16-01-31.
 */
public interface CanBeStoredAsItem {

    void attachItemData();

    float getItemWeight();

    void stopUpdateAndDraw();

    void resumeUpdateAndDraw();

    InventoryItem getItemData();

}

File: src/com/Torvald/Terrarum/Actors/CanBeStoredAsItem.kt
package com.Torvald.Terrarum.Actors

import com.Torvald.Terrarum.GameItem.InventoryItem

/**
 * Created by minjaesong on 16-03-14.
 */
interface CanBeStoredAsItem {

    fun attachItemData()

    fun getItemWeight(): Float

    fun stopUpdateAndDraw()

    fun resumeUpdateAndDraw()

    var itemData: InventoryItem?

}